### PR TITLE
Add plugin clipboardmon

### PR DESCRIPTION
**Pull request by skvl**

### CODE REVIEW

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by skvl** [comment]
@tklengyel is something wrong with this plugin?

**Comment by tklengyel** [comment]
@sklv I'm traveling at the moment so didn't have a chance to take a closer look at it yet.

**Comment by skvl** [comment]
OK. Good luck.

File: src/plugins/plugins.h [hunk 1]
```diff
@@ -168,25 +173,27 @@ static const char* drakvuf_plugin_names[] =
...
+    [PLUGIN_SSDTMON]      = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
+    [PLUGIN_DEBUGMON]     = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 1 },
+    [PLUGIN_DELAYMON]     = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
+    [PLUGIN_CPUIDMON]     = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 1 },
+    [PLUGIN_SOCKETMON]    = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
+    [PLUGIN_REGMON]       = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
+    [PLUGIN_PROCMON]      = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
+    [PLUGIN_BSODMON]      = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
+    [PLUGIN_CRASHMON]     = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
+    [PLUGIN_CLIPBOARDMON] = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0},
```
Comment (tklengyel) [inline src/plugins/plugins.h]: Just a minor nitpick, please add a space before the closing `}`

**Comment by skvl** [comment]
@tklengyel I think something is wrong with Travis-CI:
> The command "git submodule update --init libvmi" failed and exited with 1 during .

**Comment by tklengyel** [comment]
@skvl Yes I saw, fix is on its way.

**Comment by tklengyel** [comment]
Could you please rebase this on the latest master?

**Comment by skvl** [comment]
Done!

**Pull request merged**